android/Omnivore/app/src/main/java/app/omnivore/omnivore/ui/reader/WebReader.kt

```diff
@@ -40,23 +40,9 @@ fun WebReader(
 
   WebView.setWebContentsDebuggingEnabled(true)
 
-  val webReaderContent = WebReaderContent(
-    preferences = preferences,
-    item = params.item,
-    articleContent = params.articleContent,
-  )
-
-  val styledContent = webReaderContent.styledContent()
-  val isInDarkMode = preferences.themeKey == "Dark"
-
   Box {
     AndroidView(factory = {
       OmnivoreWebView(it).apply {
-        if (isInDarkMode) {
```

Comment:
I think I added this since the text color was coming out too light in dark mode. You might have solved that issue elsewhere.

Reply:
Yeah I think we were ignoring the background colour in web before so we had to.

Reply:
But then when we added Sepia we had to start setting the background colour.